src/components/FormModal.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  Modal, ModalDialog, ModalHeader, ModalTitle, ModalBody, ModalFooter, Button,
+} from 'react-bootstrap';
+import FormInputs from './FormInputs.jsx';
+
+function FormModal(props) {
+  if (props.show) {
+    return (
+      <Modal show={props.show} onHide={props.onHide}>
+        <Modal.Dialog>
+          <Modal.Header closeButton>
+            <Modal.Title>Product Review Title</Modal.Title>
+          </Modal.Header>
+
+          <Modal.Body>
+            <FormInputs />
+          </Modal.Body>
+          <Modal.Footer />
```

Comment:
I haven't looked into forms much so this might be totally wrong, but how are you planning on getting your form inputs out of the form inputs component?  I would imagine you would have to pass down a prop that sets state?